### ISSUE TITLE
Overkill tests failing to build on 8.10.3

### ISSUE THREAD

**Issue by cdornan** [ekmett/bound#81]
We see `Ambiguous occurrence ‘foldMap’` on stackage nightly (GHC 8.10.3).

(Though I don't yet understand why it manifested yesterday, Thursday, 2021-02-04.)

**Comment by phadej** [comment]
I guess `vector` release.

**Comment by RyanGlScott** [comment]
Thanks for the bug report! I've revised the affected Hackage versions accordingly, and I've opened #82 with a fix. I'll prepare a new Hackage release with that fix soon.

> (Though I don't yet understand why it manifested yesterday, Thursday, 2021-02-04.)

This one is my doing, actually. I deliberately [added `vector-0.12.2` to Stackage](https://github.com/commercialhaskell/stackage/pull/5854) in an attempt to flush out any remaining build errors that arise when using that version of `vector`. It looks like my plan worked!
